tests/unit/test_prestoclient.py

```diff
@@ -94,118 +94,52 @@ def test_http_answer_not_json(self, mock_response,
         type(mock_response.return_value).status =\
             PropertyMock(return_value=200)
         self.assertRaisesRegexp(ValueError, 'No JSON object could be decoded',
-                                client.execute_query, 'any_sql')
-
-    @patch.object(PrestoClient, 'get_response_from')
-    @patch.object(PrestoClient, 'get_next_uri')
-    def test_retrieve_rows(self, mock_uri, mock_get_from_uri, mock_presto_config):
```

Comment:
This means we prob don't have any unit test coverage of retrieve_rows, which is significant because we'll only catch issues (even syntax issues) at runtime, but we have pretty thorough product testing so this is probably ok.